mopro-core/examples/circom/keccak256/compile.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-#circom ./keccak256_256_test.circom --r1cs --wasm --sym --output ./target
-circom ./examples/circom/keccak256/keccak256_256_test.circom --r1cs --wasm --sym --output ./examples/circom/keccak256/target
+mkdir target
+circom ./keccak256_256_test.circom --r1cs --wasm --sym --output ./target
```

Comment:
We might want to have target dir higher up, given that this example dir isn't used a lot as self-contained folder